bundler.d/aws.rb

```diff
@@ -0,0 +1,3 @@
+group :aws do
+  gem 'fog-aws'
```

Comment:
I was thinking if fog-aws introduced some breaking change they might bump to 1.x?  Not sure..